master/internal/command/command_service.go

```diff
@@ -27,24 +27,22 @@ type commandService struct {
 	syslog   *logrus.Entry
 }
 
-// newCommandService creates a new commandService.
-func newCommandService() *commandService {
-	return &commandService{
+// SetDefaultCmdService initializes & returns a new commandService.
+func SetDefaultCmdService(db *db.PgDB, rm rm.ResourceManager) {
+	if DefaultCmdService != nil {
+		logrus.Warn(
+			"detected re-initialization of Command Service that should never occur outside of tests",
+		)
+	}
+
+	DefaultCmdService = &commandService{
```

Comment:
i'd prefer to export `commandService` and give it a proper constructor but i wouldn't block on it.